atomic_reactor/plugins/pre_resolve_remote_source.py

```diff
@@ -94,7 +94,9 @@ def run(self):
         remote_source_json = self.source_request_to_json(source_request)
         remote_source_url = self.cachito_session.assemble_download_url(source_request)
         remote_source_conf_url = remote_source_json.get('configuration_files')
-        self.set_worker_params(source_request, remote_source_url, remote_source_conf_url)
+        remote_source_content_manifest = self.cachito_session.get_icm(source_request)
+        self.set_worker_params(source_request, remote_source_url, remote_source_conf_url,
```

Comment:
I think this will also require an osbs-client update (add `remote_source_content_manifest` to user_params). It probably doesn't need to be done right away but only when the new plugin is added, is that the plan?

Ideally, `remote_source_content_manifest` should also be added here: [atomic_reactor/schemas/user_params.json](../../tree/master/atomic_reactor/schemas/user_params.json), but since all the other `remote_source_*` params are missing, it would just create an inconsistency.